app/assets/javascripts/discourse/app/lib/sidebar/base-custom-sidebar-section-link.js

```diff
@@ -68,22 +73,22 @@ export default class BaseCustomSidebarSectionLink {
   /**
    * @returns {string} CSS class for prefix
    */
-  get PrefixCSSClass() {}
+  get prefixCSSClass() {}
```

Comment:
I changed these because they were weirdly PascalCase here, but nowhere else was referring to them this way.